amp.js

```diff
@@ -41,6 +41,7 @@ createTask('build', 'build', 'build');
 createTask('bundle-size', 'bundleSize', 'bundle-size');
 createTask('caches-json', 'cachesJson', 'caches-json');
 createTask('check-analytics-vendors-list', 'checkAnalyticsVendorsList', 'check-analytics-vendors-list'); // prettier-ignore
+createTask('check-build-system', 'checkBuildSystem','check-build-system'); // prettier-ignore
```

Comment:
This line is short enough to fit in 80 chars, and doesn't need the `// prettier-ignore`.

```suggestion
createTask('check-build-system', 'checkBuildSystem', 'check-build-system');
```